sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -999,32 +999,30 @@ export class MessageReceiver extends LinkEntity {
     operation: DispositionType,
     options?: DispositionOptions
   ): Promise<any> {
-    return new Promise((resolve, reject) => {
+    const settleMessagePromise = new Promise((resolve, reject) => {
       if (!options) options = {};
       if (operation.match(/^(complete|abandon|defer|deadletter)$/) == null) {
         return reject(new Error(`operation: '${operation}' is not a valid operation.`));
       }
-      this._clearMessageLockRenewTimer(message.messageId as string);
       const delivery = message.delivery;
       const timer = setTimeout(() => {
-        this._deliveryDispositionMap.delete(delivery.id);
-
         log.receiver(
           "[%s] Disposition for delivery id: %d, did not complete in %d milliseconds. " +
             "Hence rejecting the promise with timeout error.",
           this._context.namespace.connectionId,
           delivery.id,
-          Constants.defaultOperationTimeoutInSeconds * 1000
+          10 * 1000
         );
-
         const e: AmqpError = {
           condition: ErrorNameConditionMapper.ServiceUnavailableError,
           description:
             "Operation to settle the message has timed out. The disposition of the " +
             "message may or may not be successful"
         };
         return reject(translate(e));
-      }, Constants.defaultOperationTimeoutInSeconds * 1000);
+      }, 10 * 1000);
```

Comment:
Reason to remove `Constants.defaultOperationTimeoutInSeconds`?

Reply:
See first part of https://github.com/Azure/azure-sdk-for-js/pull/6109#discussion_r344297030 
Since amqp-common uses default of 10 seconds per operation timeout for the `updateDisposition` counterpart, replicating same for here

Reply:
Changing the value drastically from 60 to 10 seconds can have other consequences. In a prior PR, we had changed the value from 20 to 60 seconds due to a customer issue. 60 seconds was also the suggested value from the service team.

I would strongly suggest not to make this change

Reply:
Updated, this would impact all of the test runs on CI - since the error ones would now take 3 times and 180 seconds longer to fail